modules/Database/Arangojs.js

```diff
@@ -110,10 +110,12 @@ class ArangoJS {
             for (const edgeId in graph.edges) {
                 const edge = graph.edges[edgeId];
                 if (edge !== null) {
-                    // eslint-disable-next-line no-underscore-dangle,prefer-destructuring
-                    edge._from = edge._from.split('/')[1];
-                    // eslint-disable-next-line no-underscore-dangle,prefer-destructuring
-                    edge._to = edge._to.split('/')[1];
+                    if (edge._from.indexOf('ot_vertices/') > -1) {
```

Comment:
Declare 'ot_veritces/' and 'ot_edges/' as class constants. Also, use ```_from.startsWith(otVertices)``` instead of ```_from.indexOf('ot_verices/')```. Suggestion, add private method ```__normalizeConnection()``` which will remove all necessary things.